src/routes.js

```diff
@@ -0,0 +1,5 @@
+export const toMovie = ({ id } = { id: ":id" }) => `/movie/${id}`;
+export const toPerson = ({ id } = { id: ":id" }) => `/person/${id}`;
```

Comment:
I think it might be better to keep our routes consistant and change the 
`/movie/${id}` and `/person/${id}` 
to 
`/movies/${id}` and `/people/${id}`,
cause every movie/person belongs to the movies/people category, just with a unique id added to the url on user click.